test-clients/src/main/java/com/hedera/services/bdd/spec/infrastructure/providers/ops/files/RandomFileInfo.java

```diff
@@ -33,16 +33,19 @@
 
 import static com.hederahashgraph.api.proto.java.ResponseCodeEnum.FILE_DELETED;
 import static com.hederahashgraph.api.proto.java.ResponseCodeEnum.INSUFFICIENT_TX_FEE;
+import static com.hederahashgraph.api.proto.java.ResponseCodeEnum.INVALID_FILE_ID;
 
 public class RandomFileInfo implements OpProvider {
 	private final ResponseCodeEnum[] permissibleCostAnswerPrechecks = standardQueryPrechecksAnd(
 			FILE_DELETED,
-			INSUFFICIENT_TX_FEE
+			INSUFFICIENT_TX_FEE,
```

Comment:
Question for my understanding. Is just `UNKNOWN` not permissible Prechecks and any other status is considered permissible? If so, umbrellaRedux test is just designed to submit transactions and not worry about response code

Reply:
No, UNKNOWN is a status after consensus process. It's not a status for precheck. UmbrellaRedux does check the response code and can tell whether a transaction is successful or not. That's why we have to provide the right status codes here.

Reply:
Understood. Thanks !